### PR TITLE
ACS extension doc updates: Handling notification deeplinks

### PR DESCRIPTION
**Pull request by rymorale**

### CODE REVIEW

File: using-mobile-extensions/adobe-campaign-standard/README.md [hunk 1]
```diff
@@ -331,3 +331,96 @@ To remove the corresponding mobile app in Campaign Standard, click **Remove from
 Deleting your mobile property in Experience Platform Launch does not automatically delete your Campaign Standard mobile app.
 {% endhint %}
 
+### Handling deeplinks included in Campaign In-App messages
+
+A deeplink can be added to in-app messages delivered from Adobe Campaign. The link can be a website URL such as https://www.adobe.com or an app link which can be used to direct the user to a specific area of your app.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/README.md]: - In the 1st sentence, please add "that" between "in-app messages" and "delivered".
-In the 2nd sentence, please add a comma between "link" and "which".

File: using-mobile-extensions/adobe-campaign-standard/README.md [hunk 2]
```diff
@@ -331,3 +331,96 @@ To remove the corresponding mobile app in Campaign Standard, click **Remove from
...
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Handling in-app message deeplinks on Android
+
+Website URL's are handled without any further action by the app developer. If an in-app message is clicked through and contains a valid URL, the device's default web browser will redirect to the URL contained in the in-app notification payload's `url` key (for fullscreen and alert messages) or `adb_deeplink` key (for local and push notifications). App links present in the notification payload can be handled by setting up URL schemes in the app. For more information about setting URL schemes for Android, see [Create Deep Links to App Content](https://developer.android.com/training/app-links/deep-linking).
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/README.md]: - In the 1st sentence, please remove the "'s" in URL's --> "URLs" and replace "without any further action..." with "without any additional action..."
- The 2nd sentence is really long, so I suggest breaking it up as follows:
"If an in-app message is clicked through and contains a valid URL, the device's default web browser redirects the user to the URL in the in-app notification payload's `url` in one of the following ways:
    * key (for fullscreen and alert messages)
    * `adb_deeplink` key (for local and push notifications)"

- The 3rd sentence is also pretty long, so I suggest rewording it like this:
"To handle app links in the notification payload, you need to set up URL schemes in the app. For more information about setting URL schemes for Android, see [Create Deep Links to App Content](https://developer.android.com/training/app-links/deep-linking)."

File: using-mobile-extensions/adobe-campaign-standard/README.md [hunk 3]
```diff
@@ -331,3 +331,96 @@ To remove the corresponding mobile app in Campaign Standard, click **Remove from
...
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### Handling alert or fullscreen notfication deeplinks on iOS
+
+The default behavior for opening URL's and app links on iOS is to use the Safari browser. This behavior is expected if the included deeplink is a web URL, but it does not allow the app to handle app links. To provide a better customer experience, the AEP SDK provides a URL handler for use with alert or fullscreen notification deeplinks.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/README.md]: - In the 1st sentence, please replace "URL's" with "URLs" and reword as follows:

- Reword this section as follows:
"To open URLs and app links on iOS, by default, use the Safari browser. If the included deeplink is a web URL, this is an expected behavior, but Safari does not allow the app to handle app links. To provide a better customer experience, the Experience Platform SDK provides a URL handler that you can use with alert or fullscreen notification deeplinks.

File: using-mobile-extensions/adobe-campaign-standard/README.md [hunk 4]
```diff
@@ -331,3 +331,96 @@ To remove the corresponding mobile app in Campaign Standard, click **Remove from
...
+	if url?.contains("campaigndemoapp://") ?? false {
+        // handle the app link (parse any data present in the app link and/or redirect to a desired area within the app)
+   	return true
+  }
+  // false is returned when the URL should be opened in Safari
+  return false
+})
+```
+
+#### Handling local or push notification deeplinks on iOS
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/README.md]: There is quite a bit of text in this section, so I recommend breaking it up like this:

#### Handling local or push notification deeplinks on iOS

When a local or push notification is clicked through, the `didReceiveNotificationResponse` instance method is called with the notification response being passed in as a parameter. For more information, see the Apple developer docs at [userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649501-usernotificationcenter?language=objc). 

The deeplink URL can be retrieved from the response object passed into the handler method. An example for retrieving the deeplink URL and loading web links is provided below. For more information about handling app links and setting URL schemes for iOS, see [Defining a Custom URL Scheme for Your App](https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app).

File: using-mobile-extensions/adobe-campaign-standard/README.md [hunk 5]
```diff
@@ -331,3 +331,96 @@ To remove the corresponding mobile app in Campaign Standard, click **Remove from
...
+
+#### Retrieving then loading the URL string from the local or push notification response 
+
+The deeplink included in the response can be loaded using the [openURL:options:completionHandler:](https://developer.apple.com/documentation/uikit/uiapplication/1648685-openurl?language=objc) instance method.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/README.md]: Please remove "included" and just say "in"

File: using-mobile-extensions/adobe-campaign-standard/README.md (hunk 2 above)
Comment (rymorale) [reply using-mobile-extensions/adobe-campaign-standard/README.md]: I've reworded the second bullet point's suggestion since the`url` key is part of the fullscreen and alert message payload.
